File: conect.java
import java.sql.*;

public class Main {
    public static void main(String[] args) {
        String url = "jdbc:mysql://seu_host/seu_banco_de_dados";
        String user = "seu_usuario";
        String password = "sua_senha";

        try {
            Connection conn = DriverManager.getConnection(url, user, password);

            Statement stmt = conn.createStatement();

            String sql = "SELECT * FROM tabela";
            ResultSet rs = stmt.executeQuery(sql);

            while (rs.next()) {
                System.out.println(rs.getString("coluna"));
            }

            rs.close();
            stmt.close();
            conn.close();
        } catch (SQLException e) {
            e.printStackTrace();
        }
    }
}

File: conect.py
import mysql.connector

config = {
    'user': 'seu_usuario',
    'password': 'sua_senha',
    'host': 'seu_host',
    'database': 'seu_banco_de_dados'
}

conn = mysql.connector.connect(**config)

cursor = conn.cursor()

cursor.execute("SELECT * FROM tabela")
result = cursor.fetchall()
print(result)
cursor.close()
conn.close()
